getting_started/setup_vm/ccf-dev.yml

```diff
@@ -3,15 +3,23 @@
     - import_role:
         name: intel
         tasks_from: sgx-psw.yml
+      when: platform == "sgx"
     - import_role:
         name: intel
         tasks_from: sgx-group.yml
+      when: platform == "sgx"
     - import_role:
         name: az_dcap
         tasks_from: install.yml
+      when: platform == "sgx"
     - import_role:
         name: openenclave
         tasks_from: binary_install.yml
+      when: platform == "sgx"
+    - import_role:
+        name: openenclave
+        tasks_from: install_host_verify.yml
```

Comment:
Looks like this file isn't included in this PR yet?

Reply:
It should have been added to `main` in https://github.com/microsoft/CCF/pull/4431/files#diff-e7e6a99610d72802a104fb456bb1b7f13c2f67a3353bfdd27854b1c4b0c8c53b

Reply:
Ah you're right, I just didn't see it in my old branch.